doc/developer/diagnostic-questions.md

```diff
@@ -20,10 +20,13 @@ various ways that can cause Materialize to behave not as expected.
 
 # Checklist
 
+Is the user reporting the issue the same user that set up Materialize?
+- If not, ensure the version check below is performed anew.
```

Comment:
Could we add emphasis that the version check should performed anew _by the reporting user_ and not the _user setting up Materialize_?

If Alice has set up Materialize and Bob is using it, then what is in the PR covers the case where Bob is reporting the issue. To cover the case where Alice or Charles is reporting the issue based on what Bob said to them, we might as well add to the checklist something like:
"Is the user reporting the issue the same user that is running Materialize? If not, try to get in touch with the user actually running Materialize or get readouts of what the user running Materialize is actually seeing in the client."